etl/definitions.py

```diff
@@ -1,30 +1,29 @@
 from pathlib import Path
 
 from dagster import Definitions, EnvVar, load_assets_from_modules
+from langchain_community.vectorstores.utils import DistanceStrategy
 
-from etl.models.types.enrichment_type import EnrichmentType
+from etl.models.types import EnrichmentType
 
 from . import assets
 from .jobs import embedding_job, retrieval_job
-from .resources import (
-    InputDataFilesConfig,
-    OpenaiPipelineConfig,
-    OpenaiSettings,
-    OutputConfig,
-)
+from .resources import InputConfig, OpenaiPipelineConfig, OpenaiSettings, OutputConfig
 
 openai_settings = OpenaiSettings(openai_api_key=EnvVar("OPENAI_API_KEY").get_value(""))
 
 definitions = Definitions(
     assets=load_assets_from_modules([assets]),
     jobs=[embedding_job, retrieval_job],
     resources={
-        "input_data_files_config": InputDataFilesConfig.from_env_vars(
+        "input_config": InputConfig.from_env_vars(
             data_files_directory_path_default=Path(__file__).parent.absolute()
             / "data"
             / "input"
             / "data_files",
             data_file_names_default=("mini-wikipedia.output.txt",),
+            distance_strategy_default=DistanceStrategy.EUCLIDEAN_DISTANCE,
+            score_threshold_default=0.5,
+            etl_base_iri_default="https://etl/",
```

Comment:
Use pyoxigraph tiny types instead of strings.

Reply:
I am using pyoxigraph NamedNode elsewhere, but I have removed the `etl_base_iri` and related tiny types.